internal/executor/instance/persistentworker/isolation/vetu/vetu.go

```diff
@@ -190,6 +194,10 @@ func (vetu *Vetu) WorkingDirectory(projectDir string, dirtyMode bool) string {
 }
 
 func (vetu *Vetu) Close(ctx context.Context) error {
+	if vetu.resourceModifier != nil {
+		vetu.resourceModifier.Unlock()
```

Comment:
should it be deferred? So once the VM is closed aka stopped we release the resource? Otherwise there might be a race.

Reply:
Good catch, added `defer` in https://github.com/cirruslabs/cirrus-cli/pull/764/commits/ce9524a187415698dec64064ec9f5b7e2f214300.